Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -615,8 +617,8 @@ public static class Loader {
                                     (e as patch_Entity).EntityData = entityData;
                                     return e;
                                 };
-                                if (!TypeHelper.entityDataNameToType.ContainsKey(id))
-                                    TypeHelper.entityDataNameToType[id] = type;
+                                if (!TypeHelper.EntityDataName_to_Type.ContainsKey(id))
+                                    TypeHelper.EntityDataName_to_Type[id] = type;
```

Comment:
the `ContainsKey` here is redundant (applies to other places in this method as well)

Reply:
nvm, it is slightly different. `TryAdd` would be equivalent I think

Reply:
Was unaware that was a method lol